lib/redbreast/version.rb

```diff
@@ -1,3 +1,3 @@
 module Redbreast
-  VERSION = '1.4.2'
+  VERSION = '1.4.3'
```

Comment:
Increase the minor version, this is not a hotfix.

```suggestion
  VERSION = '1.5.0'
```

Reply:
sorry, read minor correctly but for some reason increased the patch 😄